MRN-Code/coinstac
# Owners Add Members

# Problem
* Closes #455 
# Solution
* Typeahead added to allow owner to search user names
* Table changed to use checkboxes denoting user/member
* GraphQL endpoints created to grab individual user or all users
   * Individual user endpoint not yet used
# Testing
* Login and try to add a user and make that user an owner.
* Login as the new owner and make sure that you have owner privs on the consortium.
